Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Docker image for Drone's terraform deployment plugin
+#
+#     CGO_ENABLED=0 go build -a -tags netgo
+#     docker build --rm=true -t plugins/drone-terraform .
+
+FROM gliderlabs/alpine:3.2
+RUN apk-install ca-certificates
+
+ENV TERRAFORM_VERSION 0.6.9
+
+RUN apk update && \
+    wget -q "https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk" && \
+    apk add --allow-untrusted glibc-2.21-r2.apk && \
```

Comment:
I can't seem to remember exactly why this was a problem, but I'm pretty sure I didn't just copy it from somewhere (that I didn't have it and terraform wouldn't run on Alpine Linux and then I found references that were adding it (https://github.com/AlbanMontaigu/docker-terraform/commit/3e6dad67744ce3c1c6153fea0442d47add892521))